scripts/cli/builder/Dockerfile

```diff
@@ -18,6 +18,8 @@ FROM ${DEPENDENCIES_IMAGE}
 RUN apt-get install -y maven
 RUN apt-get install -y git
 RUN apt-get install -y python3
+RUN apt-get install -y pip
+RUN apt install -y python3.10-venv
```

Comment:
The apt command is designed for human use and isn't stable, this should probably use apt-get.